src/app/components/dialogs/mapSettingsDialog/mapSettingsDialog.component.html

```diff
@@ -0,0 +1,34 @@
+<app-base-dialog [data]="data">
+  <mat-drawer-container class="example-container" autosize>
+    <mat-drawer #drawer [opened]="true" mode="side">
+      <mat-selection-list #optionList [multiple]="false">
+        <mat-list-option [selected]="true" value="general">General</mat-list-option>
+        <mat-list-option value="advanced">Advanced</mat-list-option>
+      </mat-selection-list>
+    </mat-drawer>
+
+    <form class="example-sidenav-content" [formGroup]="settingsForm">
+      <div *ngIf="optionList.selectedOptions.selected[0]?.value === 'general'" class="color-box">
+        <p>General</p>
+        <mat-selection-list #colourGradientList formControlName="generalColourSelection"
+          [(ngModel)]="generalSelectionValue" [multiple]="false">
```

Comment:
Using a form control and an ngmodel here seems like overkill.  Better to use just one.